test/drenv/providers/lima/k8s.yaml

```diff
@@ -120,6 +120,28 @@ provision:
       EOF
       systemctl restart containerd
 
+  # Pre-pull kubeadm images to improve probing performance
+  # See <https://github.com/lima-vm/lima/pull/3082>
+  - mode: system
+    script: |
+      #!/bin/bash
+      set -eux -o pipefail
+      test -e /etc/kubernetes/admin.conf && exit 0
+      command -v kubeadm >/dev/null 2>&1 || exit 0
```

Comment:
This seems wrong - since we pull the images with kubeadm, and this exit if kubeadm exist, we will never pull the images in this step, and they are always pulled during kubeadm init.